db/migrate/20190808134440_create_newsfeeds.rb

```diff
@@ -0,0 +1,10 @@
+class CreateNewsfeeds < ActiveRecord::Migration[5.2]
+  def change
+    create_table :newsfeeds do |t|
+      t.string :title
```

Comment:
we should probably add database validations here
```suggestion
      t.string :title, null: false
```

Reply:
rebased into 767a80bca6c968c97783fbce30aa180b4dcacfb7